src/main/java/org/trigon/hardware/simulation/SingleJointedArmSimulation.java

```diff
@@ -6,10 +6,23 @@
 import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
 import org.trigon.hardware.RobotHardwareStats;
 
-
+/**
+ * A class that represents a simulation of a single jointed arm mechanism.
+ */
 public class SingleJointedArmSimulation extends MotorPhysicsSimulation {
     private final SingleJointedArmSim armSimulation;
 
+    /**
+     * Creates a new SingleJointedArmSimulation.
+     *
+     * @param gearbox          The motor used to control the arm
```

Comment:
All of these need to be `motor(s)`

Reply:
No, and arm doesn't necacerily have more then one motor

Reply:
That's why motor`(s)` it can be one or more. The way it's worded currently, it's only one motor